Poker.BE/Poker.BE.Domain/Game/Deck.cs

```diff
@@ -6,22 +6,28 @@
 
 namespace Poker.BE.Domain.Game
 {
-    public class Deck
+	/// <summary>Created Deck of Card and shuffle cards UC</summary>
+	/// <remarks>
+	/// <author>Gal Wainer</author>
```

Comment:
👍 Great documentation! Keep it up! I hope the rest of the team will do the same...